crab/ib-run-pr-crab.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -ex
+
+trap report EXIT
+
+report() {
+   exit_code=$?
+   if [ ${exit_code} -ne 0 ]; then
+       echo "FAILED" > $WORKSPACE/crab/statusfile
+   fi
+}
+
+if [ "${SINGULARITY_IMAGE}" = "" ] ; then
+  osver=$(echo ${SCRAM_ARCH} | tr '_' '\n' | head -1 | sed 's|^[a-z][a-z]*||')
+  ls /cvmfs/singularity.opensciencegrid.org >/dev/null 2>&1 || true
+  IMG_PATH="/cvmfs/singularity.opensciencegrid.org/cmssw/cms:rhel${osver}"
+  if [ ! -e "${IMG_PATH}" ] ; then
+    IMG_PATH="/cvmfs/unpacked.cern.ch/registry.hub.docker.com/${DOCKER_IMG}"
+  fi
+  export SINGULARITY_IMAGE="${IMG_PATH}"
+fi
+
+for CRABCLIENT_TYPE in $(ls ${PR_CVMFS_PATH}/share/cms/ | grep -Eo '(dev|prod|pre)')
+do
+    echo $CRABCLIENT_TYPE
+
+    [ "${CRABCLIENT_TYPE}" != "" ]   || export CRABCLIENT_TYPE="prod"
+    [ "${BUILD_ID}" != "" ]          || export BUILD_ID=$(date +%s)
+    [ "${WORKSPACE}" != "" ]         || export WORKSPACE=$(pwd) && cd $WORKSPACE
+
+    # Report PR status
+    source $WORKSPACE/cms-bot/pr_testing/setup-pr-test-env.sh
+    source $WORKSPACE/cms-bot/common/github_reports.sh
+    mark_commit_status_all_prs 'crab-'${CRABCLIENT_TYPE} 'pending' -u "${BUILD_URL}" -d "Running"
+    echo "CRAB is sourced from:"
+    which crab-${CRABCLIENT_TYPE}
+
+    export CRAB_REQUEST="Jenkins_${CMSSW_VERSION}_${SCRAM_ARCH}_${BUILD_ID}"
+    voms-proxy-init -voms cms
+    crab-${CRABCLIENT_TYPE} submit -c $(dirname $0)/task.py
```

Comment:
I think we should create a new clean CMSSW dev area here otherwise crab is going to use PR dev area which might have a lot of cmssw libs which crab will uselessly upload to crab server